examples/limitrange/README.md

```diff
@@ -1,4 +1,147 @@
+pod from being permanently unscheduled to a node, the operator instead chooses to reject pods that exceed 2GB
+of memory as part of admission control.
+2. A cluster is shared by two communities in an organization that runs production and development workloads
+respectively.  Production workloads may consume up to 8GB of memory, but development workloads may consume up
+to 512MB of memory.  The cluster operator creates a separate namespace for each workload, and applies limits to
+each namespace.
+3. Users may create a pod which consumes resources just below the capacity of a machine.  The left over space
+may be too small to be useful, but big enough for the waste to be costly over the entire cluster.  As a result,
+the cluster operator may want to set limits that a pod must consume at least 20% of the memory and cpu of their
+average node size in order to provide for more uniform scheduling and to limit waste.
+
+This example assumes that you have a functional Kubernetes setup.
+
+This example demonstrates how limits can be applied to a Kubernetes namespace to control
+min/max resource limits per pod.  In addition, this example demonstrates how you can
+apply default resource limits to pods in the absence of an end-user specified value.
+
+For a detailed description of the Kubernetes resource model, see [Resources](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/resources.md)
+
+Step 1: Create a namespace
+-----------------------------------------
+This example will work in a custom namespace to demonstrate the concepts involved.
+
+Let's create a new namespace called limit-example:
+
+```shell
+$ cluster/kubectl.sh create -f namespace.yaml
+$ cluster/kubectl.sh get namespaces
+NAME            LABELS             STATUS
+default         <none>             Active
+limit-example   <none>             Active
+```
+
+Step 2: Apply a limit to the namespace
+-----------------------------------------
+Let's create a simple limit in our namespace.
+
+```shell
+$ cluster/kubectl.sh create -f limits.yaml --namespace=limit-example
+```
+
+Let's describe the limits that we have imposed in our namespace.
+
+```shell
+Name:   limits
+Type    Resource  Min Max Default
+----    --------  --- --- ---
+Pod   memory    6Mi 1Gi -
+Pod   cpu   250m  2 -
+Container memory    6Mi 1Gi 100Mi
+Container cpu   250m  2 250m
+```
+
+In this scenario, we have said the following:
+
+1. A container in a pod may consume between 6Mi and 1Gi of memory.  If the container does not
+specify an explicit resource limit, each container in a pod will get 100Mi of memory.
+2. A container in a pod may consume between 250m and 2 cores of cpu.  If the container does
+not specify an explicit resource limit, each container in a pod will get 250m of cpu.
+3. The total memory usage of a pod across all of its containers must fall between 6Mi and 1Gi.
+4. The total cpu usage of a pod across all of its containers must fall between 250m and 2 cores.
```

Comment:
It would be clearer if these statements were in the same order as the line-items above; i.e. pod/mem, pod/cpu, container/mem, container/cpu.